dynosaur/tests/pass/basic-no-ret.stdout

```diff
@@ -8,6 +8,7 @@ use dynosaur::dynosaur;
 trait MyTrait {
     async fn foo(&self);
 }
+#[allow(non_snake_case)]
```

Comment:
I'd remove the commit that include all these things. We should just generate `_dynosaur_macro_dynmytrait` or something like that.